Marlin/Configuration.h

```diff
@@ -3170,6 +3170,8 @@
   #if ENABLED(TFT_COLOR_UI)
     //#define SINGLE_TOUCH_NAVIGATION
   #endif
+
+//#define TOUCH_USE_ENCODER_TOO         // Use encoder or buttons control together with touchscreen
```

Comment:
If the enabled LCD has both touch and encoder, the encoder should behave like it does currently (enabled).

Reply:
You mean default state in config? It's purely matter of taste. And check is for defined pins, not encoder actually existing, so default disabled is logical conclusion.

Reply:
> You mean default state in config? It's purely matter of taste. And check is for defined pins, not encoder actually existing, so default disabled is logical conclusion.

This adds unnecessary complexity/config options then.

We support several touch screens that also have an encoder. The existing behavior for those TFTs is to support both touch & encoder, so this should be enabled by default.

Reply:
Native touchscreens with encoder? What are those? Extui ones are not affected.

Reply:
> Native touchscreens with encoder? 

Yes. `MKS_TS35_V2_0`, `BTT_TFT35_SPI_V1_0`, `BIQU_BX_TFT70` to name a few.